.github/workflows/integration-tests.yml

```diff
@@ -72,9 +76,9 @@ jobs:
       - name: Run integration tests - multiple peers - pytest
         run: poetry -C tests run pytest tests/consensus_tests --durations=10
         timeout-minutes: 60
-      - name: upload logs in case of failure
+      - name: upload logs
         uses: actions/upload-artifact@v4
-        if: failure() || cancelled()
+        if: always()
```

Comment:
Could you elaborate why we want to have logs even from successful runs?

Reply:
I needed logs from successful run to debug my failing runs, and it's super annoying to tweak the workflow and push when you need them. The logs are just 1.8mb, I think it should be fine to always upload them? We can also reduce retention period, I don't think anyone would ever need logs from more than 10 days ago.

Reply:
I don't care too much about it, though. Just thought I'll see how everyone else feels about this.

Reply:
If we reduce the retention time, then LGTM. I just don't want to store everything and generate a humongous storage "just because".

Reply:
My brother in Christ, I've just reduced rust caches size like 10 times (like 10+ GB) each dependency bump. We have space for logs. 😂